wiki/Project_Loved/en.md

```diff
@@ -28,6 +28,7 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - Its creator is neither banned nor restricted
 - At least one difficulty can be passed
 - It has at least 30 favourites (only applies to osu!standard maps)
+- Every difficulty of a beatmapset must have a minimum [drain time](/wiki/Gameplay/Drain_time) of 30 seconds.
```

Comment:
Oh, I did not realise that. I think it should be phrased a bit differently to match the above points. Maybe something like this?
```suggestion
- Every difficulty has at least 30 seconds of [drain time](/wiki/Gameplay/Drain_time)
```
If you don't think so, then I'll go ahead and merge.

Reply:
Sure, it looks more uniform that way.